README.md

```diff
@@ -6,20 +6,21 @@ ocfweb
 
 [The main ocf website.](https://www.ocf.berkeley.edu/)
 
-
 ## Working on `ocfweb`
 
 Clone the repo, and be sure to check out submodules:
 
     $ git clone git@github.com:ocf/ocfweb.git
     $ git submodule update --init
 
+If you run into problems with scss where it can't import bootstrap, it's
+because you forgot to run the second command.
 
 ### Running in development mode
 
-Either on supernova, or on your own staff VM, run `make dev`. The first time
-will take a while, but future runs will be almost instant thanks to
-[pip-faster](https://github.com/Yelp/pip-faster).
+Either on supernova, or on your own staff VM<sup>[0]</sup>, run `make dev`.
```

Comment:
Yeah, I wondered about that, maybe it could be in a block under the paragraph or something? I guess I'm just not a fan of footnotes, but if they seem like the best way then that's fine too.